tests/unit/service_test.py

```diff
@@ -926,7 +927,7 @@ def test_get_container_data_volumes(self):
         volumes, _ = get_container_data_volumes(container, options, ['/dev/tmpfs'], [])
         assert sorted(volumes) == sorted(expected)
 
-    def test_get_container_data_volumes_image_is_none(self):
+    def test_get_container_data_volumes_image_does_not_exist(self):
```

Comment:
@shin- Here's the updated test, which currently fails.

Reply:
Mmmh ok, I'll take a look over the next couple of days. Thank you!